diff --git a/sample-storybook/src/stories/BigBox.stories.tsx b/sample-storybook/src/stories/BigBox.stories.tsx
index 181cf8b..99fec9e 100644
--- a/sample-storybook/src/stories/BigBox.stories.tsx
+++ b/sample-storybook/src/stories/BigBox.stories.tsx
@@ -12,8 +12,8 @@ type Story = StoryObj<typeof Button>
 export const ABigBox: Story = {
   render: () => 
     <div style={{
-        width: "400px",
-        height: "400px",
+        width: "900px",
+        height: "900px",
         backgroundColor: "black",
       }} 
     />,
