diff --git a/sample-storybook/src/stories/Button.stories.tsx b/sample-storybook/src/stories/Button.stories.tsx
index f49be6a..7c379c9 100644
--- a/sample-storybook/src/stories/Button.stories.tsx
+++ b/sample-storybook/src/stories/Button.stories.tsx
@@ -10,7 +10,7 @@ export default {
 
 type Story = StoryObj<typeof Button>
 
-export const Primary: Story = {
+export const StoryshotsTargetPrimary: Story = {
   render: () => <Button label="Button" primary={true} />,
 }
 
